packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -733,6 +734,7 @@ export interface CallCompositeStrings {
     soundLabel: string;
     spokenLanguageStrings?: SpokenLanguageStrings;
     starRatingAriaLabel: string;
+    starRatingCancelButtonAriaLabel: string;
```

Comment:
does survey need two different cancel button aria labels? could this just be surveyCancelButtonAriaLabel and remove TagsSurveyCancelButtonAriaLabel

Reply:
same with cancel and submit, instead just have surveyCancelButtonLabel and surveySubmitButtonLabel

Reply:
Those strings are for 2 different surveys, users might want to have different strings for confirm button and cancel button on those surveys?

Reply:
Ah yes sorry you are correct! I wrote this before I realized the star survey flows into this survey and it isn't one or the other